Web/starter_project_iconics/styles/globals.css

```diff
@@ -1,7 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;600;700;800&family=Poppins:ital,wght@0,200;0,300;0,400;0,700;0,800;1,600&display=swap');
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
+  html {
+    font-family: 'Poppins', sans-serif;
```

Comment:
is this the case for everyone ?
you can check the designs to make sure this applies for every page

Reply:
personally, i would prefer if there is a way to only do configurations in tailwind and it's own text styles - same way we can override the primary colors and so on (not sure if that is an option i think it is worth checking)